README.md

```diff
@@ -1,8 +1,39 @@
 # Minecraft Server Stack Launcher Module - MCS2LM
 Core component for launching minecraft servers.
 
+## System requirements
+
+- UNIX/Linux OS
+- docker
+- id
+- make
+
 ## Development
 
 ### Githooks
 There are certain githooks provided to ensure code quality. These are located in `scripts/githooks`.  
 In order to install githooks run `make setup-githooks` 
+
+### Set up the development environment
+The setup process will create a volume for permanent storage and also builds the official docker image used for development. The volume is used to store the application once it has been built.
+
+To set up the development environment, please run `make setup`.
+
+### Build the application
+The build process will mount the volume to `/app-bin/` and builds the binary to the same path.
+
+To build the application, please run `make build`.
```

Comment:
What is the application exactly? Please be more specific, we build the cli tool to test the features provided by this package. This also requires to build from the directory `cmd`.